tests/unit/utils/txn_parser/transaction_jsons/offer_partially-filled_and_filled.json

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
Can you remove the hyphen from the filename and rename it to `offer_partially_filled_and_filled.json` so the casing stays consistent?